src/content/overview/backstage-portal/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Backstage portal
```

Comment:
IMO, I will call it a developer portal cause it is more than a Backstage instance; we serve some special plugins, we operate it,...

Reply:
we dont use in any action the name of the tooling but what general one

Reply:
All right, fixed to align